front/src/app/core/home-page/home-page.component.html

```diff
@@ -0,0 +1,5 @@
+@if(isLogin){
+    <app-is-login-home-page></app-is-login-home-page>
+}@else{
+    <app-is-logout-home-page></app-is-logout-home-page>
```

Comment:
suggestion: tu peux éviter d'avoir 2 tags 
```suggestion
    <app-is-login-home-page/>
}@else{
    <app-is-logout-home-page/>
```